src/main/java/core/basesyntax/ProcessEnterList.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+import java.util.List;
+
+public class ProcessEnterList {
+    private final OperationsStrategyImpl operationsStrategyImpl;
+
+    public ProcessEnterList(OperationsStrategyImpl operationsStrategyImpl) {
+        this.operationsStrategyImpl = operationsStrategyImpl;
+    }
+
+    public String workWithEnterList(List<String> enter) {
+        StringBuilder str = new StringBuilder();
+
+        for (String enetr : enter) {

```

Comment:
Typo in variable name: `enetr` should be renamed to `entry` for better readability and to avoid confusion.